### DISCUSSION TITLE
1 command per second: jumpy speedup / slowdown

### DISCUSSION THREAD

**Discussion by Armi1P** [Rem0o/FanControl.Releases#1659] [Ideas]
Since FanControl only sends commands to the fan controllers every second, some devices, like pumps are audibly jumpy when the Step Up / Step Down values are set high. For example the ASUS AISuite software has slow ramping, but no matter how aggressive I set it, it always does it smoothly with no audible jumps.
Would it be possible to have a setting for changes per second?

**Comment by Rem0o** [comment]
Technically yes, but that would require some big changes under the hood, and drastically increase the resource usage.
